.eslintrc.js

```diff
@@ -63,6 +63,12 @@ module.exports = {
 			files: [ 'bin/**/*', 'test/**/*' ],
 			...nodeConfig,
 		},
+		{
+			files: [ '**/test/**/*' ],
+			rules: {
+				'react/display-name': 'off',
+			},
+		},
```

Comment:
Touching [`client/hosting/server-settings/test/index.js`](https://github.com/Automattic/wp-calypso/pull/103218/files#diff-c6c5896cc24d56d2862dc29192e933b61225cbf0c50b70518f05bb51a5943df5) causes an ESLint error, because the file mocks a component without providing a usable `displayName`.

https://github.com/Automattic/wp-calypso/blob/f7fdc1c331e750860041eca9dfdf67bbb8c804b3/client/hosting/server-settings/test/index.js#L6-L8

This rule was recently added in #103078. Given that its primary benefit is for browser debuggability, I don't think it's as critical to enforce for test files. (cc @mcsf)